src/rss/feed/Blog.tsx

```diff
@@ -6,7 +6,7 @@ import { truncateOnSpace } from './utils';
 
 export default function Blog({ item }: { item: BlogItem }) {
   return (
-    <div className={and(style.card, style['card--blog'])}>
+    <a className={and(style.card, style['card--blog'])}  href={item.url} target="_blank" rel="noopener noreferrer">
```

Comment:
Not harming anyone to include it, but I actually think noopener is default on _blank links in most/all browsers as per spec now (https://html.spec.whatwg.org/#following-hyperlinks)

Reply:
Resolved